.github/workflows/hatch_ci.yml

```diff
@@ -310,43 +310,14 @@ jobs:
           fi
 
       - name: Run Whitesource Scan
-        if: fromJson(env.has_whitesource_config) && fromJson(env.not_fork_pr)
+        if: fromJson(env.has_whitesource_config) && fromJson(env.not_fork_pr) && fromJson(env.is_push)
         uses: ./workflows-repo/.github/actions/whitesource-scan
         with:
           whitesource_product_name: ${{ inputs.whitesource_product_name }}
           whitesource_project_name: ${{ env.ws_project_name }}
           whitesource_api_key: ${{ secrets.WHITESOURCE_API_KEY }}
           whitesource_config_file: ${{ inputs.whitesource_config_file }}
 
-      - name: Run WhiteSource PR Checker (Policy Violations)
-        id: whitesource-pr-policy-violation-checker
-        if: fromJson(env.not_fork_pr) && fromJson(env.is_pr) && fromJson(env.has_whitesource_config)
-        uses: docker://ghcr.io/solacedev/maas-build-actions:latest
-        env:
-          WS_API_KEY: ${{ secrets.WHITESOURCE_API_KEY }}
-          WS_PRODUCT_NAME: ${{ inputs.whitesource_product_name }}
-          WS_PROJECT_NAME: ${{ env.ws_project_name }}
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          AWS_REGION: ${{ secrets.MANIFEST_AWS_REGION }}
-          AWS_ACCESS_KEY_ID: ${{ secrets.MANIFEST_AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.MANIFEST_AWS_SECRET_ACCESS_KEY }}
-          WS_BLOCKING_POLICY_VIOLATION_LEVELS: "Major,Minor"
-          FAIL_IF_POLICY_VIOLATIONS_FOUND: "True"
-          GH_ORG: ${{ github.repository_owner }}
-          GH_REPO: ${{ github.event.repository.name }}
-          GH_SHA: ${{ github.event.pull_request.head.sha }}
-          PR_NUMBER: ${{ github.event.pull_request.number }}
-          INCLUDE_HELPFUL_LINKS: "False"
-          STATUS_CHECK: "True"
-        with:
-          entrypoint: /bin/sh
-          args: >
-            -c "
-            . $VIRTUAL_ENV/bin/activate &&
-            cd $ACTIONS_PATH/whitesource-pr-checker &&
-            python whitesource_pr_checker.py
```

Comment:
@artyom-morozov Would you know the difference between `whitesource_pr_checker.py` and `whitesource_policy_violation_checker.py` used [here](https://github.com/SolaceDev/solace-public-workflows/blob/main/.github/workflows/hatch_ci.yml#L347C20-L347C59)

I can't seem to find these python scripts.  

I'd like to confirm the `hatch_ci` workflow covers the policy check.